### DISCUSSION TITLE
进退 | /cn/2021/12/to-be-not-to-be/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1544] [Comments]
疲惫的身躯
依然承受着驱赶蹂踩
枯干的脑海
不由思索进退的障碍
[&hellip;] 前进一步是悬崖
后退一步是荆棘
奔赴虚空的悬崖永久深埋
或钻回荆棘再杀出血路来
[&hellip;] 疲惫的身躯
枯干的脑海
已没有气力辩解是非黑白
于是一再权衡　身上最后两条安全带

https://yihui.org/cn/2021/12/to-be-not-to-be/

**Comment by ZaiyanHere** [comment]
「谁说的荆棘丛中下脚易、月明帘下转身难？荆棘丛中下脚也不易啊」🙊

**Reply by yihui** [reply]
所以这位捂嘴猴 🙊 只是装样子捂嘴？嘴捂住了，手却把字敲了出来。

话说回来，时隔两年后，我觉得这句话更有道理了，确实是月明帘下转身更难。

**Reply by ZaiyanHere** [reply]
我这不是看到了了不得的东西一时激动么 下次应该再隐晦一点儿 对个暗号儿～
